graph/GuidelinesGraph.md

```diff
@@ -166,7 +166,8 @@ Microsoft Graph APIs should support basic query options in conformance with ODat
 |Requirements                                                                                        |
 |----------------------------------------------------------------------------------------------------|
 | :heavy_check_mark: **MUST** support `$select on resource` to enable properties projection. |
-| :ballot_box_with_check: **SHOULD** support `\$filter with eq`, `ne` operations on properties of entities for collections. |
+| :ballot_box_with_check: **SHOULD** support `$expand=navProp1` option for navigation properties of entities. |
+| :ballot_box_with_check: **SHOULD** support `$filter with eq`, `ne` operations on properties of entities for collections. |
```

Comment:
```suggestion
| :ballot_box_with_check: **SHOULD** support `$filter` with `eq` and `ne` operations on properties of entity collections. |
```
Question - do we think `ne` is an important operation?  For me I wonder whether `lt` and `gt` are more important for DateTime properties.